packages/libs/eda/src/lib/core/components/computations/plugins/correlationAssayMetadata.tsx

```diff
@@ -161,81 +162,169 @@ export function CorrelationAssayMetadataConfiguration(
     });
   }, []);
 
+  // Content for the expandable help section
+  const helpContent = (
+    <div className={cx('-HelpInfoContainer')}>
+      <H6>What is correlation?</H6>
+      <p>
+        The correlation between two variables (taxa, genes, sample metadata,
+        etc.) describes the degree to which their presence in samples
+        co-fluctuate. For example, the Age and Shoe Size of children are
+        correlated since as a child ages, their feet grow.
+      </p>
+      <p>Here we look for correlation between:</p>
+      <ol>
+        <li>
+          Abundance of taxa at a given taxonomic level or functional data such
+          as pathway abundance
+        </li>
+        <li>
+          Continuous metadata variables that are compatable, i.e. on a 1-1 path
+          from the taxonomimc assay entity.
```

Comment:
I'm not sure this is clear but I'm also not sure how to improve it

Reply:
i'll give it a shot